server.js

```diff
@@ -15,27 +17,36 @@ app.get('/api/v1/tasks', (req, res) => {
   res.status(200).json({ tasks: app.locals.tasks });
 });
 
+app.get('/api/v1/savedtasks', (req, res) => {
+  res.status(200).json(app.locals.savedTasks);
+});
+
 app.get('/api/v1/tasks/:category', (req, res) => {
-  let categoryKeys = Object.keys(app.locals.tasks)
-  let arr = []
   let category = req.params.category
-  categoryKeys.forEach(key => {
-    app.locals.tasks[key].forEach((task) => {
-      let lowercase = task.category
-      let findWhiteSpace = lowercase.replace(/ /g, '')
-      lowercase = findWhiteSpace.toLowerCase()
-      if (lowercase === category) {
-        arr.push(task)
-      }
-    })
-  })
-  if (arr.length !== 0) {
-    res.status(200).json(arr)
+  if (req.params.category === 'mentalcare'){
+    category = 'mentalCare'
+  }
+  const foundTasks = app.locals.tasks[category.toLowerCase()]
+
+  if (foundTasks.length) {
+    res.status(200).json(foundTasks)
   } else {
     res.sendStatus(404)
   }
-})
+});
 
+app.post('/api/v1/savedtasks', (req, res) => {
+  app.locals.savedTasks.push(req.body);
+
+  res.status(201).json(app.locals.savedTasks);
+});
+
+app.delete('/api/v1/savedtasks', (req, res) => {
+  const { id } = req.body;
+  const filteredList = app.locals.savedTasks.filter(task => task.id !== id)
+  
```

Comment:
we'd want to alter the existing savedTasks array no?
I feel like this is not actually deleting anything within the savedTasks.
it's creating a new variable and returning that as a response (without changing our data). 
(maybe we can send back a response with just the task that was deleted instead of the filtered list)?

Reply:
thats a great point, thank you! I changed it to now reassign the savedList to the filtered one